### PR TITLE
Micro fixes

### PR DESCRIPTION
**Pull request by peppsac**
2 fix commits:

* fix(core): assign the correct layer property
This fixes a few places in the code missed during PR #758.

* fix(core): make sure PlanarControls call notifyChange when needed
This one fixes issue #776

### CODE REVIEW

**Review by zarov — COMMENTED** [review]
Thanks for those fixes, I'm sorry for not catching them before approbation ! About the `package-lock`, couldn't we update in another PR or at least commit ? Something like a `chore`

File: package-lock.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
       "integrity": "sha1-x1K9IQvvZ5UBtsbLf8hPj0cVjMQ=",
       "dev": true,
       "requires": {
-        "acorn": "^4.0.3"
+        "acorn": "4.0.13"
```
Comment (gchoqueux) [inline package-lock.json]: you could remove all changes on this file

**Comment by peppsac** [comment]
The package-lock is a mistake: reverting this :)

File: package-lock.json (hunk 1 above)
Comment (peppsac) [reply package-lock.json]: Yep, fixed.

**Comment by zarov** [comment]
There are some missing, if I'm not mistaken:
- https://github.com/iTowns/itowns/blob/master/src/Core/Prefab/PlanarView.js#L57
- https://github.com/iTowns/itowns/blob/master/src/Core/View.js#L655
- https://github.com/iTowns/itowns/blob/master/src/Core/Prefab/GlobeView.js#L113

I'll let you confirm and fix those !

**Comment by peppsac** [comment]
> I'll let you confirm and fix those !

Fixed, thanks.

**Review by zarov — APPROVED** [review]

**Pull request merged**